src/storage/src/hummock/compactor/mod.rs

```diff
@@ -98,19 +98,21 @@ impl TableBuilderFactory for RemoteBuilderFactory {
     }
 }
 
+#[derive(Clone)]
```

Comment:
Is it possible for this context to contain `compaction_filter` ?

Reply:
It is difficult because it is just a trait, rather than `dynamic type`.